lib/util/sslreq.c

```diff
@@ -1,6 +1,10 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 
+#ifdef __linux__
```

Comment:
For portability, make this (and the call which uses it later) `#ifndef SO_NOSIGPIPE`.

Reply:
Still need to fix this.